### PR TITLE
feat: catwalk

### PR DESCRIPTION
**Pull request by DakshG07**
A small tool to easily generate repo thumbnails.

This PR also fixes some of the broken hyperlinks.

### CODE REVIEW

File: catwalk/main.py [hunk 1]
```diff
@@ -70,27 +54,48 @@ def round_mask(image, radius=40):
...
+    final = Image.new("RGBA", (w, h), (0, 0, 0))
+    for i, arg in enumerate(imgs):
+        masked = gen_masked(arg, masks[i])
+        final.paste(masked, (0, 0), masked)
+
+	# put it on a coloured background, if `--background` is passed
+    if args.background:
+        m = args.margin
+
+        final = round_mask(final, args.radius)
```
Comment (nekowinston) [inline catwalk/main.py:87]: maybe we should add an "inner radius" setting as well? currently both radii are the same value

**Comment by nekowinston** [comment]
Once the CLI is sorted out, and the docs are updated after that decision, we can happily merge.

File: catwalk/main.py (hunk 1 above)
Comment (DakshG07) [reply catwalk/main.py:105]: Yeah. I would say add an optional `outer-radius` parameter, which defaults to the normal `radius`.

**Comment by nekowinston** [comment]
LGTM

**Pull request merged**